### PR TITLE
pass key to isMatch

### PR DESCRIPTION
**Pull request by someone235**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ module.exports = function filterValues(obj, filter, options) {
   var res = {};
 
   forOwn(obj, function (val, key) {
-    if (isMatch(val)) {
+    if (isMatch(val,key)) {
```
Comment (doowb) [inline index.js]: add a space after the comma here

File: test.js [hunk 2]
```diff
@@ -17,6 +17,11 @@ describe('.filter()', function () {
       return value === 'b';
     });
     o.should.eql({ b: 'b' });
+
+    o = filter({a: 'a', b: 'b', c: 'c'}, function (value,key) {
```
Comment (doowb) [inline test.js]: add a space after the comma here

File: index.js (hunk 1 above)
Comment (doowb) [reply index.js]: also, pass `obj` as the third argument

**Review by doowb — CHANGES_REQUESTED** [review]
Thanks for the PR! A couple of small changes, then I think this is good to merge.

**Comment by someone235** [comment]
I hope this is OK now.

**Review by doowb — APPROVED** [review]

**Comment by doowb** [comment]
Looks great, thanks for the PR!

**Comment by doowb** [comment]
@someone235 this has been published as 0.4.1, thanks again.

**Pull request merged**